scanners/nuclei/templates/nuclei-update-cache-job.yaml

```
@@ -4,7 +4,11 @@
 
 # We use a persistent volume for central storing of all nuclei-templates to prevent downloading it for each scan again, enabled by default.
 {{ if .Values.nucleiTemplateCache.enabled }}
+{{- if semverCompare ">=1.21-0" .Capabilities.KubeVersion.GitVersion -}}
```

```suggestion
{{- if semverCompare ">=1.21-0" .Capabilities.KubeVersion.Version -}}
```
Seems to be deprecated ... see https://github.com/helm/charts/issues/20918

---

Ah didn't know about that 👍
`helm create` still outputs a chart which uses the old syntax, I took it from there